.yarn/patches/@metamask-network-controller-npm-23.5.0-56b87eea93.patch

```diff
@@ -1,18 +1,11 @@
 diff --git a/PATCH.txt b/PATCH.txt
 new file mode 100644
-index 0000000000000000000000000000000000000000..376255036ca54b83a3f3c3e0277c2666473df30e
---- /dev/null
-+++ b/PATCH.txt
-@@ -0,0 +1,4 @@
-+We remove lookupNetwork from initializeProvider in the network controller to prevent network requests before user onboarding is completed.
```

Comment:
We should preserve this PATCH.txt file in the patch. It documents the reason for the patch existing

Reply:
Yes I am doing this, I mistakenly removed it. when I wanted just to remove the extra+ ahead of line